packages/postcss-colormin/types/index.d.ts

```diff
@@ -6,5 +6,5 @@ export = pluginCreator;
  */
 declare function pluginCreator(config?: Record<string, boolean>): import('postcss').Plugin;
 declare namespace pluginCreator {
-    const postcss: true;
+    let postcss: true;
```

Comment:
Small change in TypeScript output, seems correct as we just assign this property to the `pluginCreator` object so it is in fact mutable.